reports/PREOPS-prompt-error-msgs.ipynb

```diff
@@ -135,6 +136,10 @@
    "source": [
     "pipeline = Pipeline.from_uri(f\"$AP_PIPE_DIR/pipelines/{instrument}/ApPipe.yaml\")\n",
     "pipeline.addConfigOverride(\"diaPipe\", \"apdb.db_url\", \"dummy\")\n",
+    "\n",
+    "# For w32 and later:\n",
```

Comment:
Technically this is for `w.2024.18` or later.  So should be the very next RSP recommended stack update.